### DISCUSSION TITLE
ONNX file format specifications

### DISCUSSION THREAD

**Discussion by lubian64** [onnx/onnx#5645] [Q&A] [answered]
I need to save and read an ONNX file from a C/C++ program.
Is it possible to have or where can I find the technical specifications of the format with which the information in an ONNX file is saved?

**Comment by justinchuby** [comment, marked as answer]
You may find the spec on https://onnx.ai/onnx/repo-docs/IR.html and the protobuf format spec'ed at https://github.com/onnx/onnx/blob/main/onnx/onnx.proto3
